<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки календарных отклонений</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Праздники</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CCalendarWidget" name="calendarHoliday">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="firstDayOfWeek">
          <enum>Qt::Monday</enum>
         </property>
         <property name="gridVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CTableView" name="tableHoliday">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Переносы</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CCalendarWidget" name="calendarChangeday">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="firstDayOfWeek">
          <enum>Qt::Monday</enum>
         </property>
         <property name="gridVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CTableView" name="tableChangeday">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>577</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>F4 правка</string>
     </property>
     <property name="shortcut">
      <string>F4</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>F9 вставка</string>
     </property>
     <property name="shortcut">
      <string>F9</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header location="global">library.TableView</header>
  </customwidget>
  <customwidget>
   <class>CCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header location="global">library.calendar</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>calendarHoliday</tabstop>
  <tabstop>tableHoliday</tabstop>
  <tabstop>calendarChangeday</tabstop>
  <tabstop>tableChangeday</tabstop>
  <tabstop>buttonEdit</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>calendarHoliday</sender>
   <signal>currentPageChanged(int,int)</signal>
   <receiver>Dialog</receiver>
   <slot>updateTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarChangeday</sender>
   <signal>currentPageChanged(int,int)</signal>
   <receiver>Dialog</receiver>
   <slot>updateTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableHoliday</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>updateCalendar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableHoliday</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>updateCalendar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableHoliday</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableChangeday</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>updateCalendar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableChangeday</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>updateCalendar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableChangeday</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEdit</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
